Task2/js/app.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombieElement = document.createElement('div');
+
+        zombieElement.classList.add('zombie');
+
+        zombieElement.addEventListener('click', () => {
+            zombieElement.classList.add('dead');
```

Comment:
well... this solution is close to completed
but I guess if you click on the first zombie the last one will dead

Reply:
? but last zombie is alive... because every zombieElement have his own addEventListener.